core/flow/rule_manager_test.go

```diff
@@ -373,3 +373,27 @@ func Test_buildRulesOfRes(t *testing.T) {
 		assert.True(t, tcs[3].boundStat == stat4)
 	})
 }
+
+func TestLoadRules(t *testing.T) {
+	t.Run("loadSameRules", func(t *testing.T) {
+		_, err := LoadRules([]*Rule{
+			{
+				Resource:               "some-test",
+				Threshold:              10,
+				TokenCalculateStrategy: Direct,
+				ControlBehavior:        Reject,
+			},
+		})
+		assert.Nil(t, err)
+		ok, err := LoadRules([]*Rule{
+			{
+				Resource:               "some-test",
+				Threshold:              10,
+				TokenCalculateStrategy: Direct,
+				ControlBehavior:        Reject,
+			},
+		})
+		assert.Nil(t, err)
+		assert.False(t, ok)
+	})
+}
```

Comment:
How about add assert that `LoadRules` does return `true` when rules updated.

Reply:
Other ut has done a lot of validation for cases that return true